### DISCUSSION TITLE
Ignoring reverted commits

### DISCUSSION THREAD

**Discussion by Subbeh** [renovatebot/renovate#22412] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.32.0

### If you're self-hosting Renovate, select which platform you are using.

Gitea

### If you're self-hosting Renovate, tell us what version of the platform you run.

1.19.3

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have Renovate with auto merge enabled which works great. 
The only problem I have is whenever there is a release that causes issues and I need to revert the version upgrade, how do I tell Renovate to ignore that particular version and only upgrade subsequent versions?

As for now I have manually added the package to the `renovate.json` file, but I was hoping there was some way to avoid this.

### What you tried so far.

Add a package rule to the config

**Comment by viceice** [comment]
if you use PR merge, that renovate will detect the merged PR and blocks further updates with same version.

**Reply by rarkins** [reply, marked as answer]
Actually Renovate should now reopen merged PRs - see #22279

The answer is: If you merge a PR, revert it, and then get a replacement PR, then either ignore or close that new one. You won't get a replacement for closed PRs - only for merged.
